postgres/src/main/java/org/jdbi/v3/pg/Jsr310ArgumentFactory.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.pg;
+
+import java.lang.reflect.Type;
+import java.sql.Types;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+
+import org.jdbi.v3.StatementContext;
+import org.jdbi.v3.argument.Argument;
+import org.jdbi.v3.argument.ArgumentFactory;
+import org.jdbi.v3.argument.ObjectArgument;
+
+public class Jsr310ArgumentFactory implements ArgumentFactory {
```

Comment:
Jackson has started renaming all their classes from `Jsr310` to `JavaTime` now that it's actually part of the JDK.  Should we follow their lead?